templates/home.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'layout.html' %}
+
+{% block title %}
+    Sign Language Translator
+{% endblock %}
+
+{% block content %}
+<div class="containers-wrapper">
+    <div id="container">
+        <img id="videoElement" />
+        <button id="startButton">Start Webcam</button>
+        <button id="stopButton">Stop Webcam</button>
+    </div>
+    
+    <div class="right-aligned-container">
+        <img src="/static/ASL_chart.jpg" alt="ASL Chart" id="aslChart">
+        <textarea rows="10" cols="30" readonly id="transcriptionTextarea"></textarea>
+    </div>
+</div>
+
+<script>
```

Comment:
Preferred: Like your css styling, make a separate folder and file to hold your javascript logic.